error: expected identifier
 --> tests/ui/error-unsupported-struct-attr-format.rs:4:13
  |
4 | #[frompyo3( = "Resource")]
  |             ^
